core/arch/arm/plat-imx/platform_config.h

```diff
@@ -128,8 +129,16 @@
 #define GICD_OFFSET			0x1000
 #define GIC_CPU_BASE			(GIC_BASE + GICC_OFFSET)
 #define GIC_DIST_BASE			(GIC_BASE + GICD_OFFSET)
+
+#if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
+	defined(PLATFORM_FLAVOR_mx6qsabresd)
 #define UART1_BASE			0x02020000
 #define UART2_BASE			0x021E8000
+#else
+#define UART1_BASE			0x02020000
```

Comment:
I thought it was better this way, so UART definitions are grouped by SoC family, but I can get UART1_BASE out of the ifdef.